### PR TITLE
Refactor filter helper funtion

### PR DESCRIPTION
**Pull request by Marina20V**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @andrii-kuzmichov 
- [ ] @VadymKharyton 

### Second Level Review

- [ ] @github_username

## Summary of issue

refactor and lint fix component

## Summary of change

refactoring code in the component 

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/helpers/filterHelper.js [hunk 1]
```diff
@@ -3,32 +3,28 @@ import { stringify as queryStringStringify } from "query-string";
 const filterHelper = (() => {
   return {
     isObject(object) {
-      return object !== null && typeof object === "object";
+      return !object && typeof object === "object";
```
Comment (andrii-kuzmichov) [inline src/components/helpers/filterHelper.js:6]: is it correctly condition?

File: src/components/helpers/filterHelper.js [hunk 2]
```diff
@@ -3,32 +3,28 @@ import { stringify as queryStringStringify } from "query-string";
...
+        const compareObjectsKeys =
+        keysObjectFirst.length !== keysObjectSecond.length;
+         
+          return (areObjects && !compareObjectsKeys) ||
           (!areObjects && valObjectFirst !== valObjectSecond)
-        ) {
-          return false;
-        }
-      }
+      });
```
Comment (andrii-kuzmichov) [inline src/components/helpers/filterHelper.js:24]: are you sure about it?

File: src/components/helpers/filterHelper.js (hunk 1 above)
Comment (Marina20V) [reply src/components/helpers/filterHelper.js]: I understood my not correct condition. Thanks.

File: src/components/helpers/filterHelper.js (hunk 2 above)
Comment (Marina20V) [reply src/components/helpers/filterHelper.js]: I understood my not correct decision to solve the issue. Thanks.

**Pull request closed without merging**